omise/omise.php

```diff
@@ -0,0 +1,44 @@
+<?php
+if (! defined('_PS_VERSION_'))
+    exit();
+
+class Omise extends PaymentModule
+{
+
+    /**
+     * The name that will be display to the user at the back-end.
+     *
+     * @var string
+     */
+    const MODULE_DISPLAY_NAME = 'Omise';
+
+    /**
+     * The name that used to referece in the program.
+     *
+     * @var string
+     */
+    const MODULE_NAME = 'omise';
+
+    /**
+     * The version of the module.
+     *
+     * @var string
+     */
+    const MODULE_VERSION = '1.6.0.0';
+
+    public function __construct()
+    {
+        $this->name = self::MODULE_NAME;
```

Comment:
@guzzilar If this style was introduced, it required the maintenance and consistency all files in this project. Although, this time there are not many files but in the future, this project will growing definitely.

Form my opinion, we are slow in the process of code review especially the point and discussion like this.

In the code review, I like to found the problem, non-function, or bug so that it can be removed before release to the user.

I encourage to use a tool to automatically check the standard of the source code such as PHP_CodeSniffer or other tools that can be run on the developer machine (at the first step) and it can reduce the time, objective and discussion in part of code review.

I strongly agree with coding standard but if it slow the time to market down, it must be carefully considered.